axelor-quality/src/main/resources/views/TrackingNumberCharacteristic.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_7.4.xsd">
+
+  <grid name="tracking-number-characteristic-grid" title="Tracking number characteristics"
+    model="com.axelor.apps.quality.db.TrackingNumberCharacteristic">
+    <field name="trackingNumber"/>
+    <field name="product"/>
+    <field name="characteristic"/>
+    <field name="measuredValue" x-scale="3"/>
+    <field name="observedValue"/>
+    <field name="conforms"/>
+  </grid>
+
+  <form name="tracking-number-characteristic-form" title="Tracking number characteristic"
+    model="com.axelor.apps.quality.db.TrackingNumberCharacteristic">
+    <panel name="mainPanel">
+      <field name="trackingNumber"/>
+      <field name="product"/>
+      <field name="characteristic"
+        onChange="action-quality-method-tracking-number-characterisctic-assign-conforms"/>
+      <field name="conforms" hideIf="characteristic.characteristicTypeSelect == 3"/>
+      <field name="characteristic.characteristicTypeSelect" hidden="true" colSpan="12"/>
+      <panel name="measurablePanel" colSpan="12"
+        showIf="characteristic.characteristicTypeSelect == 1">
+        <field name="measuredValue" x-scale="3"
+          onChange="action-quality-method-tracking-number-characterisctic-assign-conforms"/>
+        <field name="characteristic.unit"/>
+      </panel>
+      <panel name="listPanel" colSpan="12"
+        showIf="characteristic.characteristicTypeSelect == 2">
+        <field name="observedValue"
+          onChange="action-quality-method-tracking-number-characterisctic-assign-conforms"/>
+      </panel>
+      <panel name="commentPanel" colSpan="12">
+        <field name="trackingNumberComment" colSpan="12"/>
+      </panel>
+    </panel>
+  </form>
+
+  <action-method
```

Comment:
Please follow the naming convention. For action, this is as follow:
<img width="1115" height="198" alt="actionNamingConvention" src="https://github.com/user-attachments/assets/e10a66eb-ec23-4ead-88dc-9d04017dc9d8" />